proto/asn1/asn1-universal-types/asn1_universal_types.proto

```diff
@@ -0,0 +1,77 @@
+// This proto represents the Universal types
+// found in X509 Certifiactes.
+syntax = "proto2";
+
+package asn1_universal_types;
+
+// See X.690 (2015), 8.1.2.5.
+enum Encoding {
```

Comment:
Is there a reason this can't just import from asn1_pdu.proto? It doesn't seem like we need two definitions here, right?

Reply:
I've taken out Encoding, since primitive is used to encode valid DER bitstrings, while constructed is used for valid BER. We agreed that for now we're going to focus on valid Der only.